// Copyright 2022 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;
option java_package = "org.chromium.chrome.browser.fast_checkout.proto";
option java_multiple_files = true;

package autofill_assistant.fast_checkout;

// Waits for user selection and communicates it back to external action.
message WaitForFastCheckoutUserSelection {}

message FastCheckoutAction {
  oneof action {
    WaitForFastCheckoutUserSelection wait_for_user_selection = 1;
  }
}
